tasks/findFeatures.js

```diff
@@ -174,33 +171,15 @@ const generateFeatures = ({ locations, report, options }) => {
         }
 
         // Check if the location exists within our provinces
-        for (let feature of provinceData.features) {
-          if (
-            (
-              location.country === feature.properties.gu_a3 ||
-              location.country === feature.properties.adm0_a3
-            )
-            &&
-            (
-              location.state && (
-                location.state === feature.properties.name ||
-                location.state === feature.properties.name_en ||
-                location.state === feature.properties.region
-              ) ||
-              location.county && (
-                location.county === feature.properties.name ||
-                location.county === feature.properties.name_en ||
-                location.county === feature.properties.region
-              )
-            )
-          ) {
+        for (const feature of provinceData.features) {
+          if ((location.country === feature.properties.gu_a3 || location.country === feature.properties.adm0_a3) && ((location.state && (location.state === feature.properties.name || location.state === feature.properties.name_en || location.state === feature.properties.region)) || (location.county && (location.county === feature.properties.name || location.county === feature.properties.name_en || location.county === feature.properties.region)))) {
```

Comment:
This is totally unreadable now, but TBH, no expression should look like this anyway. I'll split it out into variables and test those in the expression instead of slapping it all in the `if`.

Reply:
Yeah agreed, feel like the original code for this is a little unreadable, if anything this highlights it as an opportunity ;)